en/services/opendroneid.md

```diff
@@ -52,25 +80,25 @@ An example Android receiver implementation for broadcast ASTM drone ID messages
 Code related to (Internet) Network Remote ID can be found in the [InterUSS Project](https://github.com/interuss) and https://github.com/uastech/standards (Unofficial reference for UAS-related APIs).
 
 
-## Routing of Open Drone ID MAVLink messages inside the Unmanned Aircraft System (UAS)
+## Routing Drone ID MAVLink messages inside the UAS {#routing}
 
 There can be multiple components in an UAS involved in the handling of drone ID data.
 An example is shown in the figure below.
 Certainly not all UAS will contain all of these components and the placement of some of them can be different from one system to another.
 
 ![drone ID conceptual overview](../../assets/opendroneid/conceptual_overview.png)
 
-All senders of MAVLink drone ID messages must fill the `sysid` field with the system ID value that the sender component belongs to and fill the `compid` field with the component ID value of the sender.
+All senders of MAVLink drone ID messages must fill the `sysid` [field](https://github.com/ArduPilot/pymavlink/blob/master/generator/C/include_v2.0/mavlink_types.h#L115) with the system ID value that the sender component belongs to and fill the `compid` [field](https://github.com/ArduPilot/pymavlink/blob/master/generator/C/include_v2.0/mavlink_types.h#L116) with the component ID value of the sender.
```

Comment:
Not sure about these links to the generator (or to equivalents in the generated files: https://github.com/mavlink/c_library_v2/blob/master/mavlink_types.h#L115 ). The problem is that these links might move as they are on the master branch.

One option is to simply duplicate the message definition info here. Or maybe point to here: https://mavlink.io/en/guide/serialization.html#mavlink2_packet_format

Not blocking merging on this though.

Reply:
I thought about that but then I looked at the history of that file and it seems to be changing only very rarely. Of course it is still possible that something will be added and those line references then point to something nonesense.

I was not aware of that Serialization documentation page. Maybe that would be a better place to link to.